frontend/src/i18n/locales/zh/app.json

```diff
@@ -0,0 +1,51 @@
+{
+  "Failed to fetch plugin info": "",
+  "An error occurred while fetching plugin info from {{  URL }}.": "",
+  "Yes": "",
+  "No": "",
+  "Plugin Installation": "",
+  "Do you want to install the plugin \"{{ pluginName }}\"?": "",
+  "You are about to install a plugin from: {{ url }}\nDo you want to proceed?": "",
+  "About": "",
+  "Quit": "",
+  "Select All": "",
+  "Delete": "",
+  "Services": "",
```

Comment:
I wonder did you miss these ones, or is this something to be done later?

Reply:
sorry for that, I met some ci problem, and the ci tips say `if you meet the problem, you can execute npm run i18n`, and it seem that the npm task add these new translation after I execute the task

Reply:
thanks for pointing that, I even forgot the translation in `app.json` file. Already update the translation, not to much